### DISCUSSION TITLE
Results from charm without orthonormal definition

### DISCUSSION THREAD

**Discussion by MareikeGann** [simnibs/simnibs#293] [7.Creating head models]
Hello!
I am making manual edits to charm segmentations in ITK-SNAP due to electrode artifacts on T1/T2 scans. Unfortunately, in one case I can't open the T1_upsampled.nii.gz nor the tissue_labeling_upsampled.nii.gz from the label_prep folder in ITK-SNAP as I always get the following error: 'No orthonormal definition found!'. However, I can open the files in other software and I can also open the original T1.nii.gz as well as the T2_reg.nii.gz in ITK-SNAP. Do you have any idea what might have happened and how I could get charm to always give results in the format needed?
I am using charm as follows: charm ID T1 T2 --forceqform 

Thank you, 
Mareike 

![ITK-SNAP-error](https://github.com/simnibs/simnibs/assets/103769949/0c6d8e35-705f-4079-b138-8eec99ac39a2)

**Comment by oulap** [comment]
Sounds like there's something wrong with the header of the image. Could you send me the header information of the T1.nii.gz and T1_upsampled.nii.gz, e.g., using fslhd?

**Reply by MareikeGann** [reply]
Thanks for the quick reply! Here the headers:
[T1_header.txt](https://github.com/simnibs/simnibs/files/14449092/T1_header.txt)
[T1_upsampled_header.txt](https://github.com/simnibs/simnibs/files/14449094/T1_upsampled_header.txt)

**Comment by oulap** [comment]
The qform code of the upsampled scan seems to be zero, which might be a problem for ITK as it generally prefers the qform (as far as I'm aware). Can you try to change the qform code to 1 as in the T1.nii.gz scan and see if that works? I need to check how we write out the upsampled scan as apparently the codes are not respected.

**Reply by MareikeGann** [reply]
Thanks! Using 'fslorient -setqformcode 1 imagename' on the T1 and the tissue_labeling_upsampled.nii.gz did the trick and ITK-SNAP now opens them nicely.

**Reply by oulap** [reply]
Great, I'll fix this on the simnibs end.
